websites/N/Notespace/presence.ts

```diff
@@ -4,13 +4,16 @@ const presence = new Presence({
 	browsingTimestamp = Math.floor(Date.now() / 1000);
 
 const enum Assets {
-	Icon = "https://cdn.rcd.gg/PreMiD/websites/N/Notespace/assets/logo.png",
+	Icon = "https://cdn.rcd.gg/PreMiD/websites/N/notespace/assets/logo.png",
 }
 
 presence.on("UpdateData", async () => {
-	const currentPath = document.location.pathname,
+	const currentPath = document.location.href.replace(
```

Comment:
why, pathname should just work

Reply:
it dosent because when the path has /app/#/ustawienia it dosnt get any value after #